### PR TITLE
Add debug output for cfg options

### PR DESCRIPTION
**Pull request by jichenjc**
spend a couple of time in finding why the error:
Cloud provider could not be initialized: could not init
cloud provider "openstack": Authentication failed

turn out if we can log those with -v flag it will be helpful

### CODE REVIEW

**Comment by jichenjc** [comment]
I signed it

**Comment by jichenjc** [comment]
"I signed it"

File: pkg/cloudprovider/providers/openstack/openstack.go [hunk 1]
```diff
@@ -172,11 +172,26 @@ type Config struct {
 	Networking   NetworkingOpts
 }
 
+func logcfg(cfg Config) {
+	glog.V(3).Infof("AuthURL: %s", cfg.Global.AuthURL)
+	glog.V(3).Infof("Username: %s", cfg.Global.Username)
+	glog.V(3).Infof("UserID: %s", cfg.Global.UserID)
+	glog.V(3).Infof("Password: %s", cfg.Global.Password)
```
Comment (zetaab) [inline pkg/cloudprovider/providers/openstack/openstack.go]: printing password to log is not good thing

**Comment by zetaab** [comment]
with this code in this PR, this will be added to be executed always. As I see it this is not needed always. And should we really print passwords to log even in debug cases? (its not that difficult to check cloud-config file)

**Comment by jichenjc** [comment]
you are right, I removed the password stuffs

but the log output is helpful to debug as I actually spent some time in sync open rc and cloud conf and figure out the issue, if I can log it then it should be much easier

File: pkg/cloudprovider/providers/openstack/openstack.go [hunk 2]
```diff
@@ -172,11 +172,25 @@ type Config struct {
 	Networking   NetworkingOpts
 }
 
+func logcfg(cfg Config) {
+	glog.V(3).Infof("AuthURL: %s", cfg.Global.AuthURL)
```
Comment (dims) [inline pkg/cloudprovider/providers/openstack/openstack.go]: @jichenjc can we move it to `V(5)`? since `V(3)` will log always?

**Comment by dims** [comment]
@jichenjc please update with suggestion inline.

File: pkg/cloudprovider/providers/openstack/openstack.go (hunk 2 above)
Comment (jichenjc) [reply pkg/cloudprovider/providers/openstack/openstack.go]: ok, will do ,thanks for the reminder

**Comment by hogepodge** [comment]
@adisky the code changed upstream and created a merge conflict. I manually fixed the conflict with the inline github editor, which explains the additional commit (and why I'm the author of it).

**Comment by jichenjc** [comment]
should this ready to merge? seems not find something block this PR?

**Pull request merged**